docs/source/tensors.rst

```diff
@@ -170,7 +170,6 @@ view of a storage and defines numeric operations on it.
    .. automethod:: addr
    .. automethod:: addr_
    .. automethod:: allclose
-   .. automethod:: angle
```

Comment:
Do these methods in tensors.rst need to stay in? I don't need to modify them for any of my changes, so flagging in case these should stay.

Reply:
I was taking them out because I thought they were the source of my failures. However it looks like they were not. so i added another commit that puts them back in.